examples/prescriptors/zero/prescribe.py

```diff
@@ -15,7 +15,8 @@
             'C8_International travel controls',
             'H1_Public information campaigns',
             'H2_Testing policy',
-            'H3_Contact tracing']
+            'H3_Contact tracing',
+            'H6_Facial Coverings']
```

Comment:
`zero` was missing the last NPI for masks

Reply:
Can we have the NPIs in a single file in this library so they can be imported from anywhere? Even more useful now this library is pip-installable.

Reply:
We can probably refactor that, yes